apps/docs/src/guide/types/native-parameters.md

```diff
@@ -74,3 +74,11 @@ To pass an `AssetId` as an input parameter to a Sway program, you can define the
 For a Sway program that returns an `AssetId` type, you can convert the returned value to a `string` as shown below:
 
 <<< @/../../docs-snippets/src/guide/types/contract-types.test.ts#asset-id-output
+
+### `CreateAssetId`
+
+To create an `AssetId` from a `contractId` and `subId`. You can create the `assetId` as shown below:
+
+```
+ const assetId = createAssetId(contractId, subId);
+```
```

Comment:
We don't use static code for any of our documentation snippets, instead we use snippets taken from tests, like above on line `76`. This ensures it is kept up to date. Please create a test that utilises the new function and reference it in your documentation.

Reply:
Thank you very much for reviewing my pull request @danielbate. I will remove the documentation from here and extend the that of `getMintedAsset`. The `createMintedAsset` is just another way to express it.